pkg/controller/constraint/constraint_controller.go

```diff
@@ -82,6 +83,9 @@ func (a *Adder) InjectTracker(t *readiness.Tracker) {
 	a.Tracker = t
 }
 
+func (a *Adder) InjectMutationCache(mutationCache *mutation.System) {
```

Comment:
nit: make this definition one line

Reply:
Done

Reply:
still two lines?

Reply:
Forgot to push changes, should be better now